src/patternfly/components/Tabs/examples/Tabs.css

```diff
@@ -0,0 +1,26 @@
+#ws-core-c-tabs-vertical .ws-preview-html,
+#ws-core-c-tabs-box-vertical .ws-preview-html {
+  display: flex;
+  align-items: stretch;
+}
+
+.ws-lite-index-example .ws-lite-example,
```

Comment:
can you scope all of these class changes? Otherwise they'll apply to all examples.